app/package.json

```diff
@@ -56,6 +57,7 @@
     "fs-extra": "8.1.0",
     "hash-sum": "2.0.0",
     "html-minifier": "4.0.0",
+    "html-webpack-inline-source-plugin": "^0.0.10",
```

Comment:
Do these need to be in core, or can they be in @quasar/tauri ?

Reply:
For that to happen, we'll need to chainWebpack on the tauri lib side.. that's not done right now because it needs discussion about modes outside core